app/src/main/java/com/android/bookswap/data/DataBook.kt

```diff
@@ -35,3 +35,58 @@ enum class BookLanguages(val languageCode: String) {
   ROMANSH("RM"), // Romansh, a language spoken in Switzerland
   OTHER("OTHER") // All languages that are not yet implemented
 }
+
+/** List of all supported book languages */
+val LIST_BOOK_LANGUAGES =
+    listOf(
```

Comment:
It isn't a good practice to do the list manually as if in the future we need to add a new language we may forget to add to this list. This list can be remplaced by BookLanguages::class.java.enumConstants?.map { it.name }. Therefore I think this value can be removed.